Tools/autotest/arducopter.py

```diff
@@ -8839,6 +8839,14 @@ def PAUSE_CONTINUE(self):
         self.send_pause_command()
         self.wait_groundspeed(speed_min=0, speed_max=1, minimum_duration=5)
         self.send_resume_command()
+        
+        # sending a pause, or resume, to the aircraft twice, doesn't result in reporting a failure
+        self.wait_current_waypoint(wpnum=5, timeout=500)
+        self.send_pause_command()
+        self.send_pause_command()
```

Comment:
Does this test actually fail if this PR's changes aren't added?

Reply:
@rmackay9 yes it does. Without this change AP returns `MAV_RESULT_FAILED`  if the pause or resume command is send more than once